applications/nrf_desktop/src/modules/Kconfig.config

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "Settings loading service"
+
+config DESKTOP_SETTINGS_LOAD_THREAD_STACK_SIZE
```

Comment:
I didn't use config in the name because it looks strange "config settings load", I suggest to change name of the module because it loads settings from flash and uises Zephyrs settings functionality.

Reply:
Module is named `config` because it was used to store configurations for sensor module (later also for BLE QoS module). Still I don't mind renaming it.

Reply:
Config module that uses settings module to store its data ;-)